Common/Models/StatusExtensionContext.swift

```diff
@@ -274,6 +306,7 @@ struct StatusExtensionContext: RawRepresentable {
     var loop: LoopContext?
     var netBasal: NetBasalContext?
     var batteryPercentage: Double?
+    var activeInsulin: ActiveInsulinContext?
```

Comment:
you should bump the private version number above to reflect the fact that you've added a new member to the context